cmd/commands/service.go

```diff
@@ -304,22 +304,25 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 	return command
 }
 
-func ServiceDelete(fcClient *core.Client) *cobra.Command {
+func ServiceDelete(riffClient *core.Client) *cobra.Command {
 
-	deleteServiceOptions := core.DeleteServiceOptions{}
+	options := core.DeleteServiceOptions{}
```

Comment:
feedback from @ericbottard: introduce a CLI option struct that accept a slice of names